eng/pipelines/akv-official-pipeline.yml

```diff
@@ -0,0 +1,144 @@
+#################################################################################
+# Licensed to the .NET Foundation under one or more agreements.                 #
+# The .NET Foundation licenses this file to you under the MIT license.          #
+# See the LICENSE file in the project root for more information.                #
+#################################################################################
+
+name: $(Year:YY)$(DayOfYear)$(Rev:.r)
+
+# @TODO: Add triggers and schedules
+
+parameters:
+    - name: oneBranchType
+      displayName: "OneBranch template"
+      type: string
+      values:
+          - Official
+          - NonOfficial
+      default: Official
+
+    - name: buildConfiguration
+      displayName: 'Build configuration'
+      type: string
+      values:
+          - Release
+          - Debug
+      default: Release
+
+    - name: buildPreview
+      displayName: 'Build is preview'
+      type: boolean
+      default: false
+
+    - name: publishSymbols
+      displayName: 'Publish symbols'
+      type: boolean
+      default: false
+
+variables:
+    - template: /eng/pipelines/variables/common-variables.yml@self
+    - template: /eng/pipelines/variables/onebranch-variables.yml@self
+    - template: /eng/pipelines/variables/akv-official-variables.yml@self
+
+resources:
+    repositories:
+        - repository: templates
+          type: git
+          name: 'OneBranch.Pipelines/GovernedTemplates'
+          ref: 'refs/heads/main'
+
+extends:
+    template: 'v2/OneBranch.${{ parameters.oneBranchType }}.CrossPlat.yml@templates'
+
+    parameters:
+        featureFlags:
+            WindowsHostVersion: 1ESWindows2022
+
+        # @TODO: Reassess if these should be enabled for preview builds as well
+        globalSdl:
+            # See https://aka.ms/obpipelines/sdl
+
+            apiscan:
+                # @TODO: Needs to be run for AKV?
+                enabled: ${{ not(parameters.buildPreview) }}
+                softwareFolder: '${{ variables.apiScanDllPath }}'
+                softwareName: 'Microsoft.Data.SqlClient'      # @TODO: Should be separate for AKV?
```

Comment:
This name is registered with APIScan so they know what service/product the run is related to. We should leave it as-is.

Reply:
Thanks for verifying!

Reply:
Can you also comment on the SBOM package name? The guidance for that is pretty sketchy these days, and there doesn't appear to be any separate service we had to onboard for it, just a bom that gets generated and dumped in the artifacts folder.